pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+
+  lib,
+
+  buildGoModule,
+
+  fetchFromGitHub,
+
+}:
+
+buildGoModule rec {
+  pname = "gf";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = "cococolanosugar";
```

```suggestion
    owner = "gogf";
```
Please follow upstream's repo, not your fork.